draft-ietf-quic-transport.md

```diff
@@ -3987,6 +4031,9 @@ Issue and pull request numbers are listed with a leading octothorp.
 - Added PATH_CHALLENGE and PATH_RESPONSE frames to replace PING with Data and
   PONG frame. Changed ACK frame type from 0x0e to 0x0d. (#1091, #1086)
 
+- Made connection IDs symmetrical and variable length.  Removed
```

Comment:
The fact that I called this "asymmetric" and you called it "symmetrical" suggests that both names are bad. Maybe "unidirectional" is the term we want?

Reply:
I did that intentionally :)  But the changelog can be better.  I might avoid all those words and say what they are instead.